### ISSUE TITLE
Issue report for C07

### ISSUE THREAD

**Issue by shokhova-eve** [khairulhaaziq/41-test#14]
Possible issue with ex04 - memory of the result is never freed
  ✓ [1] ft_strjoin with size 0 Expected "", got ""
  ✓ [2] ft_strjoin with size 1 Expected "Hello", got "**Hello**"
      [3] ft_strjoin with size 3 Expected "Hello, world, !", got "**Hello**Hello, world, !"
      [4] ft_strjoin with size 4 and empty strings Expected "-Hello--world", got "**HelloHello, world, !**-Hello--world"
      [5] ft_strjoin with size 2 and empty separator Expected "Helloworld", got "**HelloHello, world, !-Hello--world**Helloworld"
 FAIL  ex03/ft_strjoin.c
 
I think, I used a bad way to concatenate strings.

Adding this line after allocating memory for my char *result resolved the issue. 

result = (char *)malloc(result_size);
**result[0] = '\0';**

**Comment by k11q** [comment]
Interesting discovery, do you want to create a PR? Also, I don't think that's the best solution, if possible, try lookout for other options.

**Comment by shokhova-eve** [comment]
Yes, and I can also add more tests for ex05 to cover the atoi component of the task. Also I failed the ex05 on number 0. Other test with convertation from 01234567890 to 0123456 worked. I'm on the last week of piscine, so will soon have time.

**Comment by k11q** [comment]
> Yes, and I can also add more tests for ex05 to cover the atoi component of the task. Also I failed the ex05 on number 0. Other test with convertation from 01234567890 to 0123456 worked. I'm on the last week of piscine, so will soon have time.

Sure, that would be great!
